RealmModelGenerator/EntitieViewController.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  EntitieViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/21/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+
+class EntitiesViewController: NSViewController, EntitiesViewDelegate, EntitiesViewDataSource {
+    static let TAG = NSStringFromClass(EntitiesViewController)
+    
+    @IBOutlet weak var entitiesView: EntitiesView!
+    
+    private var schema:Schema = Schema()
+    private var model: Model = Schema.init().createModel()
+    private var entities:[Entity] = []
+    private var selectedIndex = -1
```

Comment:
have selectedIndex be an optional Int instead of just Int,
lets not use -1 as a marker value for no selected index but the optional to handle that.